src/rulesets/resolver.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from '@stoplight/path';
```

Comment:
We've had trouble where "resolver" and "resolving" are overloaded in our domain, sometimes meaning de-referencing, sometimes meaning converting external refs to internal refs, etc. It looks like this is trying to find local files, could we maybe call it finder or something else thats not resolver?